lib/Mojolicious/Static.pm

```diff
@@ -39,11 +39,16 @@ sub dispatch {
 }
 
 sub file {
-  my ($self, $rel) = @_;
+  my ($self, $rel, $c) = @_;
```

Comment:
I believe this would be inconsistent with all similar APIs, where the controller is always the first argument after the invocant.